CourseGrab/ViewControllers/Home/HomeTableViewController.swift

```diff
@@ -295,6 +295,34 @@ extension HomeTableViewController {
         headerView.configure(for: tableSections[section])
         return headerView
     }
+    
+    override func tableView(_ tableView: UITableView, didHighlightRowAt indexPath: IndexPath) {
+        let cell = tableView.cellForRow(at: indexPath) as! HomeTableViewCell
+        cell.setBackgroundColor(isHighlighted: true)
+    }
+    
+    override func tableView(_ tableView: UITableView, didUnhighlightRowAt indexPath: IndexPath) {
+        let cell = tableView.cellForRow(at: indexPath) as! HomeTableViewCell
+        cell.setBackgroundColor(isHighlighted: false)
+    }
```

Comment:
@danielVebman 
Can we refactor all of this code into the HomeTableViewCell with variable properties for isSelected? 
Something like override var isSelected: Bool {
didSet {
backgroundColor = isSelected ? ... : ...
}
}